### DISCUSSION TITLE
Using assets folder instead of mods

### DISCUSSION THREAD

**Discussion by NoobManWasTaken** [Leather128/LeathersFunkinEngine#21] [Q&A] [answered]
I want to use the assets folder instead of the mods folder because I want to change some external assets, would this have any downsides?

I am doing this without source code by the way

**Comment by Leather128** [comment, marked as answer]
first: what do you mean by 'external assets'

second: you do know you can replace default game assets with a mod in the mods folder right? (it would be like adding anything else not in the base game, just using file names of the base files lol).

**Reply by Leather128** [reply]
also using the mods folder is recommended because it makes disabling / enabling mods easier to manage for people in the future.
